spec/mailers/user_mailer_spec.rb

```diff
@@ -29,7 +29,7 @@
     end
 
     it 'assigns confirmation_url for email body' do
-      @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
+      @mail.body.encoded.should match("https://localhost:3000/g/#{@group.key}")
```

Comment:
The testing environment usually comes with a blank ENV (unless you've defined a `test.env` file for dotenv, or set it in a different way). 
A couple different options for testing ENV-based behaviour [here](https://robots.thoughtbot.com/testing-and-environment-variables) (I personally much prefer caching and resetting the ENV value to stubbing the ENV object.)

```
it 'uses https when FORCE_SSL is set to true' do 
  ssl_env = ENV['FORCE_SSL']
  ENV['FORCE_SSL'] = true
  expect(@mail.body.encoded).to match /https:/
  ENV['FORCE_SSL'] = ssl_env
end

it 'uses http when FORCE_SSL is set to false' do 
  ssl_env = ENV['FORCE_SSL']
  ENV['FORCE_SSL'] = false
  expect(@mail.body.encoded).to match /http:/
  ENV['FORCE_SSL'] = ssl_env
end
```